test_cases/stdlib/builtins/check_dict.py

```diff
@@ -20,18 +20,21 @@
 class KeysAndGetItem(Generic[_KT, _VT]):
     data: dict[_KT, _VT]
 
+    def __init__(self, data: dict[_KT, _VT]):
```

Comment:
I'm surprised pyright didn't complain about the return type being unknown!

```suggestion
    def __init__(self, data: dict[_KT, _VT]) -> None:
```

Reply:
You marked this as resolved but the suggestion hasn't been applied — any reason why not? :)

Reply:
Must be a Github android app bug (been trying it over the last week), or I miskicked.